.gitignore

```diff
@@ -13,3 +13,6 @@
 !var/SymfonyRequirements.php
 /vendor/
 /web/bundles/
+/doc/*.html
+/.idea/
```

Comment:
I think that it is just more convenient for all. But I can remove it, no problem. I think we need third person to make decision :smile: